### DISCUSSION TITLE
Tooltip colors

### DISCUSSION THREAD

**Discussion by claudiux** [SebastJava/mint-themes#16] [General]
Hi @SebastJava,

I greatly appreciate all of your themes. My favorite is Mint-Yz-Darkest-Grey.

But I think the colors of the tooltips should be changed. Straw yellow, it stings the eyes!

I modified, in `cinnamon.css`:
```
 #Tooltip {
  border: 1px solid #d0d0d0;
  border-radius: 3px;
  padding: 5px 12px;
  background-color: #fbeaa0;
  color: #4a4a4a;
  font-size: 1em;
  font-weight: normal;
  text-align: center; }
```
to:
```
#Tooltip {
  border: 1px solid #d0d0d0;
  border-radius: 3px;
  padding: 5px 12px;
  background-color: #414747;
  color: #f0f0f0;
  font-size: 1em;
  font-weight: normal;
  text-align: center; }
```
The result looks much better to me, but I'll let you judge the relevance of the colors.

Regards.
Claudiux

**Comment by SebastJava** [comment]
Hi @claudiux,

So, your favorite is Mint-Yz-Darkest-Grey? I hope you don’t find the other themes too colourful. Stay tuned, a new edition is coming soon. Some base colors got subtle improvements, and most of the dark variants are now slightly subdued. That’s on my local HDD, i will push this on GitHub and make a new release when i am pretty sure these colors are all final. I am testing them one by one before pushing it all.

I was thinking about the tooltips colors. I like this straw yellow on the light themes, but i understand your point about it being too bright for the dark themes. It could be some other dark tint for the dark themes. But i have a problem here. I don’t know what to do. Sorry but i don’t like your grey #414747 because the other backgrounds are already grey. We need something that’s differentiated. That’s what’s good with the yellowish #fbeaa0 background: there is no yellow theme so these tooltips are clearly differentiated from everything else.

So I don’t know what to do with the tooltips for the dark themes. You don’t want this yellow. I don’t want this grey. I’ll keep thinking about it. When some satisfying color is found, to get things quickly automated for all the themes, changes should be made in the **src/Mint-Y/gtk-3.0/sass/_colors.scss** like this:
```
//replacing lines 56-57:
$tooltip_bg_color: if($variant =='light', #FBEAA0, #HEXVALUE);
$tooltip_fg_color: if($variant =='light', #4A4A4A, #HEXVALUE);
//replacing #HEXVALUE with some good color...
```
And then you’ll have to run `./generate-themes.py` to get things done.

**Comment by claudiux** [comment]
Hi @SebastJava 

Yes, my favorite theme is Mint-Yz-Darkest-Grey because it is the most relaxing for my eyes.

Thank you for the tips, I will try some other colors, but not too flashy.

**Comment by SebastJava** [comment]
Hi @claudiux 

It’s good to learn there are some people who cannot support "flashy" colors. I heard this before but i found it hard to believe, at first. But soon after these first negative comments, a few months ago, i searched the web and found this: https://uxmovement.com/content/why-you-should-avoid-bright-saturated-background-colors/. So there is something i can understand there. Personally, i really needed those vibrant colors to get energized and motivated. One of my favourites is Mint-Yz-Base-Orange!

It looks like half of the crowd wants "flashy" colors, and the other half do not. So i am trying to make a palette with some vibrant colors and some soft colors. Like MintSoda vs MintSoft. I am still re-editing some of those colors, again. Was it you who suggested i could make a PPA? That was a good idea. Sorry i still haven’t took the time to look at it. But that would help, as i am re-editing over and over again. But i have a good excuse to postpone this PPA adventure: it looks like most of the downloads are made directly from https://www.pling.com/p/1408266/... And, also, i am afraid i would have to transfer all this work into Launchpad.

I am not sure you will appreciate my tip on crafting the dark tooltips using the official method. I did not put much explanations, but my preceding post was based on using some part of this big process: https://github.com/SebastJava/mint-yz-theme/blob/colors/Do-It-Yourself.md. You will probably find this far too complicated for such a small tweak. Your direct edit on the CSS is okay and faster and more simple. But repetitive and boring if you want to change all the dark themes.

This whole Mint-Yz thing was created as a reaction to the washed out colours on the official Mint-Y. I really don’t like any of those colours. That’s white text on a pale, pastel, grey-beige that is vaguely tinted with colours. It turns me sad and sleepy. I think the official Mint-Y could be made of 50% highly subdued colours and 50% vibrant colours. One day, maybe. I mean having 2 or 3 different blues and greens. And one vibrant orange. And maybe one classic, subdued purple. But we don’t want to end up with many dozens of themes, so i would just get rid of those brown, sand, teal, etc.

**Comment by SebastJava** [comment]
Obviously, it is easy to get some blue that’s nice and soft and easy on the eyes, while it is impossible to create a "soft" red.

You can get a heavily subdued red but i wouldn’t call this colour "red" anymore.

**Comment by SebastJava** [comment]
@claudiux You could find this interesting: https://forums.linuxmint.com/viewtopic.php?p=1991471#p1991471

**Comment by SebastJava** [comment]
> [...] I will try some other colors, but not too flashy.

Sometime later, maybe, i should search the web and test a new color palette made by some professionals. Something in between the heavily subdued Mint-Y old color palette on one side and my colourful, «flashy» color palette. Something that could get general approval.

All the story: https://github.com/SebastJava/mint-yz-theme/discussions/15
